agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -17,7 +17,7 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 group=default
 dir="/tmp"
 sysinfo_opts_default=( block libvirt kernel_config sos topology )
-sysinfo_opts=${sysinfo_opts_default[*]}
+sysinfo_opts="${sysinfo_opts_default[*]} ara"
```

Comment:
I think what @atheurer was saying the other day was to add `ara` to the `sysinfo_opts_default` list.  Then have the `collect_ara_data` only work if it finds that it can collect ara data.

What do you think?